project3/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="style.css">
+        <title>Day Map</title>
+    </head>
+
+    <body>
+        <h1>In a Day</h1>
+        <section class="morning">
+            <svg class="makeup" xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 128 160">
```

Comment:
SVGs have classnames, but they don't appear to be used at all?
```suggestion
            <svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 128 160">
```